packages/terra-enzyme-intl/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+* Fixed
+  * Fixed jest test failure caused by react-intl update.
+
```

Comment:
should `Added load configurations for downloading screenshots` be added to the CHANGELOG?